src/sentry/workflow_engine/migration_helpers/utils.py

```diff
@@ -0,0 +1,66 @@
+from typing import cast
+
+from sentry.incidents.models.alert_rule import AlertRule, AlertRuleTrigger, AlertRuleTriggerAction
+from sentry.models.team import Team
+from sentry.users.services.user import RpcUser
+
+MAX_ACTIONS = 3
+
+
+def get_action_description(action: AlertRuleTriggerAction) -> str:
+    """
+    Returns a human readable action description
+    """
+    ACTION_TYPE_TO_STRING = {
+        AlertRuleTriggerAction.Type.PAGERDUTY.value: "PagerDuty",
+        AlertRuleTriggerAction.Type.SLACK.value: "Slack",
+        AlertRuleTriggerAction.Type.MSTEAMS.value: "Microsoft Teams",
+        AlertRuleTriggerAction.Type.OPSGENIE.value: "Opsgenie",
+    }
```

Comment:
i'd recommend pulling this out of the method definition - that way we don't have to GC the lookup table when we invoke `get_action_description`